src/applications/pact-act/containers/HomePage.jsx

```diff
@@ -34,14 +34,13 @@ const HomePage = ({ router, setIntroPageViewed }) => {
         to apply.
       </p>
       {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-      <a
+      <va-link-action
         data-testid="paw-start-form"
         href="#"
-        className="vads-u-display--block vads-u-margin-top--4 vads-c-action-link--green"
+        class="vads-u-margin-top--2"
```

Comment:
Interesting. Does className not work for non-react components?

Reply:
@eselkin For the DST components they take class as a prop for className to apply them properly. Thanks for the review!